ansible/roles/deploy-baremetal/templates/kickstart

```diff
@@ -0,0 +1,44 @@
+# Created by Ansible on {{ansible_date_time.date}} at {{ansible_date_time.time}}.
```

Comment:
I thought all of them had it. Whatever then, I don't have a strong reason for any side.
We should fix this inconsistency later.